src/lgtv_rs232/lgtv.py

```diff
@@ -179,12 +179,16 @@ class LgTV():
         },
     }
 
-    def __init__(self, path, update=True):
-        # path is a string set to the location of the serial device, 
-        # e.g. "/dev/ttyUSB0"
+    def __init__(self, path, tv_id=0, update=True):
+        # path is a string set to the location of the serial device, e.g. "/dev/ttyUSB0"
+        # id is the tv id whcich can be set in tv options, as defalut value brodcast id is used
         self._volume = None
         self._muted = None
         self._input = None
+        if 0 <= tv_id <= 100:
```

Comment:
Should this be `0 < tv_id < 100`? 100 is not a valid id, right?

Reply:
Right